includes/right-sidebar-functions.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Header template related functions
+ */
+namespace FinAid\Theme\Includes\RightSidebar;
+
+
+/**
+ * Returns right sidebar template markup.
+ *
+ * @since 1.0.0
+ * @author RJ Bruneel
+ * @return string HTML markup for the right sidebar template
+ */
+function get_right_sidebar_template_markup( $post, $parent_post ) {
+    ob_start();
+?>
+    <article class="<?php echo $post->post_status; ?> container mt-4 mt-sm-5 mb-5 pb-sm-4">
```

Comment:
Should this be an `aside` or configurable? It seems like sidebar content would normally not be the main focus of the page.